res/values/strings.xml

```diff
@@ -1022,6 +1022,12 @@
     <string name="notifications_avg_minutes">On average every %1$d minutes</string>
     <string name="notifications_avg_hours">On average every %1$d hours</string>
     <string name="last_check_at">Checked at %1$s</string>
+    <!-- iOS webxdc selector -->
+    <string name ="webxdcs">Apps</string>
+    <string name ="webxdc_empty_hint">Received or sent apps will appear here. Tap \"Files\" to select downloaded apps.</string>
+    <!-- iOS webxdc shortcut page -->
+    <string name ="shortcut_share_btn">Click the share button</string>
+    <string name ="shortcut_add_to_home_description">Select - Add to Home Screen - to add the app to your home screen.</string>
```

Comment:
```suggestion
    <string name ="shortcut_step1_tap_share_btn">Click the share button</string>
    <string name ="shortcut_step2_tap_add_to_home_screeb">Select \"Add to Home Screen\" to add the app to your home screen.</string>
```

having the "step" in the key makes it easier for translators as the strings will appear together in correct order. it is also clearer for us :)

for the quotes around "Add To Home Screen": i know, there are some isszes with them, however, if it is not possible to render this character, we should change that programmatically eg, by `replace('"', ' - ')` - i fear otherwise, some translator will use them anyway.

Reply:
the key `shortcut_step2_tap_add_to_home_screeb` has an typo.